app/views/devise/registrations/edit.html.erb

```diff
@@ -31,6 +31,11 @@
     <%= f.password_field :current_password, autocomplete: "current-password" %>
   </div>
 
+  <div class="field">
```

Comment:
Suggestion: when this page is currently updated anyway, it may be a good idea to use consistent Bootstrap styling (https://getbootstrap.com/docs/5.2/forms/overview/) with proper layout management instead of `<br />` etc.

Reply:
could you elaborate on what Bootstrap features we should use instead, and why they're superior to `<br/>`? We're new with bootstrap, so just trying to learn :)